actionpack/lib/action_dispatch/request/utils.rb

```diff
@@ -32,7 +32,7 @@ def self.check_param_encoding(params)
           unless params.valid_encoding?
             # Raise Rack::Utils::InvalidParameterError for consistency with Rack.
             # ActionDispatch::Request#GET will re-raise as a BadRequest error.
-            raise Rack::Utils::InvalidParameterError, "Non UTF-8 value: #{params}"
+            raise Rack::Utils::InvalidParameterError, "Enconding is invalid for paramter value: #{params.scrub}"
```

Comment:
👋 @arthurnn! There's some typos in the message. How about: `"Invalid encoding for parameter: #{params.scrub}"` 😊